pubpub/pubpub
# feature: Editable PubEdge component

This PR implements an editable variant of the `PubEdge` component. I have refactored the structure of the `PubEdge` into `PubEdgeLayout` for easy reuse between the two.

![image](https://user-images.githubusercontent.com/2208769/87085549-0cdebf80-c1fe-11ea-906c-d8d8085c1d72.png)

_Test plan:_
- Make sure the Storybook fixture for `PubEdgeEditor` renders and lets you edit the values
- Make sure `PubEdge` still renders as expected.
